README.md

```diff
@@ -82,7 +84,7 @@ Your answer goes here...
     <div data-ng-app="myapp"></div>
   </body>
 </html>
-```
+``` -->
 
```

Comment:
Option A is the only one that would make the could work. You want your ng-app directive to be at the top most level of your html so that it's domain is the entire document.